storage/oasis/runtime.go

```diff
@@ -87,9 +87,8 @@ func (rc *RuntimeClient) Name() string {
 
 func (rc *RuntimeClient) nativeTokenSymbol() string {
 	for _, network := range config.DefaultNetworks.All {
-		if network.ChainContext != rc.network.ChainContext {
-			continue
-		}
+		// Iterate over all networks and find the one that contains the runtime.
+		// Any network will do; we assume that paratime IDs are unique across networks.
```

Comment:
paratime IDs are chosen by the entity that registers them

Reply:
Oooh! A potential for some malarkey then. Very good to know, thanks.
I just left a TODO/note of this; once we sort out how to merge this with your https://github.com/oasisprotocol/oasis-indexer/pull/352, this hacky approach will hopefully not be needed any more.

Actually, we'll need to tweak/expand `DefaultChains` from #352 a little so that they include some runtime info. The existing `DefaultNetworks` from oasis-sdk assign a paratime to a specific chaincontext, but really with dump-restores they can be assigned to a "metachain", i.e. to all of the chaincontexts that comprise(d) mainnet.
We could do that inference programmatically, but I think it will be much more readable to just copy paste a li'l more info over into `DefaultChains` and then use it here.

Reply:
Ah actually your PR intentionally limits its scope to consensus only. Added a task to https://github.com/oasisprotocol/oasis-indexer/pull/352/files#r1152431146 where I previously started discussing follow-ups.

Reply:
yeah, and that DefaultNetworks lookup also gets complicated by the custom chains used for testing etc.